ldes-fragmentisers/ldes-fragmentisers-common/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/fragmentation/FragmentDeleterScheduler.java

```diff
@@ -0,0 +1,33 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.fragmentation;
+
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.events.fragmentation.FragmentDeletedEvent;
+import be.vlaanderen.informatievlaanderen.ldes.server.fragmentation.entities.Fragment;
+import be.vlaanderen.informatievlaanderen.ldes.server.fragmentation.repository.FragmentRepository;
+import org.springframework.context.ApplicationEventPublisher;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Service;
+
+@Service
+public class FragmentDeleterScheduler {
```

Comment:
Same, personal preference but FragmentDeletionScheduler would be more fluent

Reply:
updated